### DISCUSSION TITLE
Comment section in Podia

### DISCUSSION THREAD

**Discussion by mcharlie9** [jacekschae/learn-pedestal-course-files#4] [Feedback]
Just a side note: I noticed you added the comment section in Podia for this course which is nice, but if this discussion page turns out to work well as a forum alongside the repo (all in one place), then keeping another comment section in podia may be redundant and time consuming for you to care for two separate avenues. Of course, either way works; my suggestion is simply to keep only one of them eventually. They are both nice forums for students to ask questions and exchange ideas. The conversations stay there (just like Slack public conversations) for all students to search and benefit from.

**Comment by jacekschae** [comment]
I guess the thing in Podia is created automatically. I didn't added that, let me see if I can turn it off. This is great -- thanks!
